acceptance/openstack/sharedfilesystems/v2/messages/messages.go

```diff
@@ -0,0 +1,34 @@
+package messages
+
+import (
+	"testing"
+
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/openstack/sharedfilesystems/v2/messages"
+)
+
+// DeleteMessage will delete a message. An error will occur if
+// the message was unable to be deleted.
+func DeleteMessage(t *testing.T, client *gophercloud.ServiceClient, message *messages.Message) {
+	err := messages.Delete(client, message.ID).ExtractErr()
+	if err != nil {
+		t.Fatalf("Failed to delete message %s: %v", message.ID, err)
+	}
+
+	t.Logf("Deleted message: %s", message.ID)
+}
+
+// PrintMessage will print a message and all of its attributes.
+func PrintMessage(t *testing.T, message *messages.Message) {
```

Comment:
Can `tools.PrintResource` be used instead? If you really want to see the `CreatedAt` and `ExpiresAt` times, you should be able to do `tools.PrintResource(t, message.CreatedAt)`.

Reply:
Done